packages/twenty-front/src/modules/localization/utils/isRtl.ts

```diff
@@ -0,0 +1,7 @@
+import { i18n } from '@lingui/core';
+import { isRtlLocale } from 'twenty-shared/utils';
+
+/**
+ * Returns whether the currently active i18n locale is a RTL locale.
+ */
+export const isRtl = (): boolean => isRtlLocale(i18n.locale);
```

Comment:
this is unlikely to behave well in react:
- either we have a function like this and we will re-render everytime we call it (as react cannot identify that the return has not changed)
- ==> I feel we should have a context at the top of the app that's set once we get the locale from the user (or default to the browser one if we don't have it yet)